src/Tribe/RSVP.php

```diff
@@ -468,6 +518,69 @@ public function generate_tickets( ) {
 	}
 
 	public function send_tickets_email( $order_id ) {
+		$all_attendees = $this->get_attendees_by_transaction( $order_id );
+		$to_send = array();
+
+		if ( empty( $all_attendees ) ) {
+			return;
+		}
+
+		// Look at each attendee and check if a ticket was sent: in each case where a ticket
+		// has not yet been sent we should a) send the ticket out by email and b) record the
+		// fact it was sent
+		foreach ( $all_attendees as $single_attendee ) {
+			// Do not add those attendees/tickets marked as not attending (note that despite the name
+			// 'qr_ticket_id', this key is not QR code specific, it's simply the attendee post ID)
+			if ( 'yes' !== get_post_meta( $single_attendee[ 'qr_ticket_id' ], self::ATTENDEE_RSVP_KEY, true ) ) {
+				continue;
+			}
+
+			// Only add those attendees/tickets that haven't already been sent
+			if ( empty( $single_attendee[ 'ticket_sent' ] ) ) {
+				$to_send[] = $single_attendee;
+				update_post_meta( $single_attendee[ 'qr_ticket_id' ], self::ATTENDEE_TICKET_SENT, true );
+			}
+		}
+
+		/**
+		 * Controls the list of tickets which will be emailed out.
+		 *
+		 * @param array $to_send        list of tickets to be sent out by email
+		 * @param array $all_attendees  list of all attendees/tickets, including those already sent out
+		 * @param int   $order_id
+		 *
+		 */
+		$to_send = (array) apply_filters( 'tribe_tickets_rsvp_tickets_to_send', $to_send, $all_attendees, $order_id );
+
+		if ( empty( $to_send ) ) {
+			return;
+		}
+
+		// For now all ticket holders in an order share the same email
+		$to = $all_attendees['0']['holder_email'];
```

Comment:
Hmm, seems like a harmless change to make but on balance I think I will leave alone, just in case we cause breakages on the first Tuesday following a new moon when quoted indicies turn out to be needed :cold_sweat: